cucumber-expressions/go/cucumber_expression.go

```diff
@@ -78,83 +94,140 @@ func (c *CucumberExpression) Source() string {
 	return c.source
 }
 
-func (c *CucumberExpression) processEscapes(expression string) string {
-	return ESCAPE_REGEXP.ReplaceAllString(expression, `\$1`)
+func (c *CucumberExpression) escapeRegexAndJoin(expression []token, prefix string, suffix string) string {
+	var builder strings.Builder
```

Comment:
`strings.Builder` is only available since golang 1.10, don't we support earlier versions?

Reply:
golang is a new language and it's not widely used. So I think it makes sense to only support new(er), versions. But I don't fully use it yet, so maybe I'm not the best person to have an opinion

Reply:
In that case, maybe we should specify a min supported version somewhere on the docs (if there isn't already any).

Reply:
@luke-hill Go is 10 years old and I'd argue it's [widely used](https://www.tiobe.com/tiobe-index/)

That said I think we should be explicit about what Go versions are supported, and I think it's fine to say >= 1.10 until somebody complains about it.